e2e/test/scenarios/sharing/public-question.cy.spec.js

```diff
@@ -140,6 +136,64 @@ describe("scenarios > public > question", () => {
       });
     }),
   );
+
+  it("should be able to view public questions with snippets", () => {
+    cy.visit("/");
+
+    openNativeEditor();
```

Comment:
`openNativeEditor` already does `cy.visit("/")` under the hood

```suggestion
    openNativeEditor();
```